types/theme.d.ts

```diff
@@ -242,3 +244,12 @@ export interface IFontFace {
 }
 
 export type FontFaces = IFontFace[]
+
+// ========================================================
+// SVG Icons
+// ========================================================
+
+type Classes = { [iconPart: string]: string }
```

Comment:
This seems to be like a general type that can be reused in some other places. Should we change the definition with something like `type Classes = { [componentPart: string]: string }` and place it in the utils section? If not, maybe we should change the name to IconClasses in order to be more specific.

Reply:
yes, thanks - let me make this move if we'll agree on the general approach

Reply:
Note, this type exists in this file already:

https://github.com/stardust-ui/react/blob/e460060d318cbe3018a48f4625dfc9b263625b12/types/theme.d.ts#L107